fictadvisor-api/src/v2/utils/documentation/pageText/getAllTexts.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiDocumentationParams } from '../decorators';
+import { PageTextsResponse } from '@fictadvisor/utils';
+
+export const PageTextDocumentationGetAllTexts: ApiDocumentationParams = {
```

Comment:
Name of the documentation object (PageTextDocumentationGetAllTexts) must be the same as corresponding controller method (PageTextDocumentationGetAll)